components/sidebar.js

```diff
@@ -90,6 +91,42 @@ const Link = styled.a`
   }
 `;
 
+const MiddleLink = styled.a`
```

Comment:
I'm thinking we can remove the `margin-top` from `Link`, and add a `margin-bottom` to `HeaderContainer` instead? Tbh I'm not sure if the Logout button needs a margin on top of it?